docs/narr/introduction.rst

```diff
@@ -394,506 +423,80 @@ without conflict with other translations in other domains.
 
 Example: :ref:`i18n_chapter`.
 
-HTTP caching
-~~~~~~~~~~~~
+Build efficient applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Pyramid provides an easy way to associate views with HTTP caching policies. You
-can just tell Pyramid to configure your view with an ``http_cache`` statement,
-and it will take care of the rest::
+Pyramid provides an easy way to *cache* the results of slow or expensive views.
+You can indicate in view configuration that you want a view to be cached::
```

Comment:
use ``.. code-block:: python`` You might need to remove the trailing dots after the ``def()``.